test/integration/integration.cc

```diff
@@ -396,4 +396,12 @@ void BaseIntegrationTest::sendRawHttpAndWaitForResponse(int port, const char* ra
   connection.run();
 }
 
+IntegrationTestServerPtr
+BaseIntegrationTest::createIntegrationTestServer(const std::string& bootstrap_path,
```

Comment:
Does it make sense to put `TestEnvironment::temporaryFileSubstitution` call here and take `json_path` instead? It would simplify the call site.

Reply:
It's got several call sites, most of which don't use TestEnvironment::temporaryFileSubstitution (mostly in legacy_json_integration_test.cc).  So I don't think that'll work.  Let me know if you see an option I don't.

Reply:
I think this is good.